daniel-v2/README.md

```diff
@@ -27,10 +28,18 @@
   - ✅✨ [Sitemap](https://wpdeveloperstaging.wordpress.com/docs/platform-features/sitemaps/)
   - ✅✨ [Jetpack Scan](https://wpdeveloperstaging.wordpress.com/docs/platform-features/jetpack-scan/)
   - ✅✨ [Account Security](https://wpdeveloperstaging.wordpress.com/docs/platform-features/account-security/)
+  - 🚧 [Automatic Data Center Failover](https://github.com/Automattic/dotcom-content/issues/218)
+  - 🚧 [Distributed Denial of Service (DDOS) and Web Application Firewall (WAF) protection](https://github.com/Automattic/dotcom-content/issues/219)
+  - 🚧 [Jetpack Search](https://github.com/Automattic/dotcom-content/issues/220)
+  - 🚧 [Akismet Anti-Spam](https://github.com/Automattic/dotcom-content/issues/221)
+  - 🚧 [Jetpack SEO Tools](https://github.com/Automattic/dotcom-content/issues/222)
+  - 🚧 [Cloudflare Integration](https://github.com/Automattic/dotcom-content/issues/223)
+  - 🚧 [Jetpack High-Level Walkthrough](https://github.com/Automattic/dotcom-content/issues/225)
 - Troubleshooting
   - ✅✨ [Monitoring](https://wpdeveloperstaging.wordpress.com/docs/troubleshooting/monitoring/)
   - ✅✨ [Debugging](https://wpdeveloperstaging.wordpress.com/docs/troubleshooting/debugging/)
   - ✅✨ [Jetpack Activity Log](https://wpdeveloperstaging.wordpress.com/docs/troubleshooting/jetpack-activity-log/)
+  - 🚧 [Resolve Jetpack Errors](https://github.com/Automattic/dotcom-content/issues/224)
```

Comment:
While I am sure this doc will be helpful, I feel that the title creates negative emotions around Jetpack. Maybe changing just the title to "Resolve common errors" or "FAQ"?

Reply:
That is a very good point I also discussed with @gcsecsey. This might sound like "Jetpack is problematic". The content of the document specifically covers Jetpack, so "Resolve common errors" could be too generic; 

I would suggest not including it here and leaving it in Support as it currently is; what do you think, @gcsecsey and @cpapazoglou ?

Reply:
I'm OK with leaving it in Support too. Yes, the problems the support page details are Jetpack specific, so I don't think we could remove it from the title.